webpack/ForemanTasks/Components/TasksTable/TasksTable.scss

```diff
@@ -0,0 +1,6 @@
+.tasks-pagination {
+  margin-top: -6px;
+}
```

Comment:
can you add an empty line here?
`[stylelint] Expected empty line before rule (rule-empty-line-before) [rule-empty-line-before]`

Reply:
Added thanks